.travis.yml

```diff
@@ -0,0 +1,104 @@
+# Travis CI (MIT License) configuration file for Options Importer
+# @link https://travis-ci.org/
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Specify when Travis should build.
+branches:
+  only:
+    - master
+    - /^release-v.*$/
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - ./vendor
+
+matrix:
+  include:
+    - php: '5.3'
+      env: WP_VERSION=3.8
+      dist: precise
+    - php: '5.3'
+      env: WP_VERSION=latest
+      dist: precise
+    - php: '5.6'
+      env: WP_VERSION=3.8
+    - php: '7.0'
+      env: WP_TRAVISCI=phpcs
+    - php: '7.0'
+      env: WP_VERSION=latest
+    - php: '7.3'
+      env: WP_VERSION=latest
+    - php: '7.4'
+      env: WP_VERSION=latest
+    - php: '7.4'
+      env: WP_VERSION=nightly
+  fast_finish: true
+
+# Prepare your build for testing.
+# Failures in this section will result in build status 'errored'.
+before_script:
+  # Turn off Xdebug. See https://core.trac.wordpress.org/changeset/40138.
+  - phpenv config-rm xdebug.ini || echo "Xdebug not available"
+  - export OG_DIR="$(pwd)"
+
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
+
+  # Couple the PHPUnit version to the PHP version.
+  - |
+    case "$TRAVIS_PHP_VERSION" in
+      7.*)
+        echo "Using PHPUnit 6.1"
+        composer global require "phpunit/phpunit=6.1.*"
+        ;;
+      *)
+        echo "Using PHPUnit 4.8"
+        composer global require "phpunit/phpunit=4.8.*"
+        ;;
+    esac
+
+  - |
+    if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
+      # Composer Install
+      travis_retry composer install
+      export PATH=$PATH:`pwd`/vendor/bin/
+    fi
+
+  - phpenv rehash
+
+  # For debugging.
+  - pwd
+  - which phpunit
+  - phpunit --version
+  - echo $PATH
+
+# Run test script commands.
+# Default is specific to project language.
+# All commands must exit with code 0 on success. Anything else is considered failure.
+script:
+  # Search for PHP syntax errors.
+  #
+  # Only need to run this once per PHP version.
+  - |
+    if [[ "$PHP_LINT" == "1" ]] ; then
+      find . -type "f" -iname "*.php" -not -path "./vendor/*" | xargs -L "1" php -l
+    fi
+
+  # WordPress Coding Standards.
+  #
+  # These are the same across PHP and WordPress, so we need to run them only once.
+  #
+  # @link https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+  # @link http://pear.php.net/package/PHP_CodeSniffer/
+  - |
+    if [[ "$WP_PHPCS" == "1" ]] ; then
+      phpcs -n
+    fi
+
```

Comment:
Does this need another script to run PHPUnit if WP_VERSION is present? This is what I'm seeing in other configs:

```
  - |
    if [[ ! -z "$WP_VERSION" ]] ; then
      phpunit
      WP_MULTISITE=1 phpunit
    fi
```

Reply:
Oh, uh, actually, looks like we don't have any tests 🙈 so we don't really need this. Not sure we're really getting any value out of these jobs if the code is not being booted.

Reply:
Tests will be coming in a future PR :)